src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -151,6 +151,13 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		startupCommand = common.ExtendPathForCommand(startupCommand, gen.SourcePath)
 	}
 
+	logger.LogInformation("Looking for appinsights loader and to export it to NODE_OPTIONS if needed")
+	scriptBuilder.WriteString("if [ -f ./" + common.ManifestFileName + " ]; then\n")
```

Comment:
Where is the manifest file being sourced? I think the one in line 62 is behind a conditional check, so it would be available always. I think we should always source it if the manifest file is available and we should do this source at the very beginning (may be after the `set -e` command i think)